lib/api/src/import_object.rs

```diff
@@ -145,20 +145,6 @@ impl IntoIterator for ImportObject {
 
 impl fmt::Debug for ImportObject {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        enum SecretOption {
-            None,
-            Some,
-        }
-
-        impl fmt::Debug for SecretOption {
-            fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-                match self {
-                    Self::None => write!(f, "None"),
-                    Self::Some => write!(f, "Some(...)"),
-                }
-            }
-        }
-
```

Comment:
This change seems entirely unrelated. Not sure if you intended to commit it

Reply:
It's deadcode, raising a warning from `rustc`.